flixel/system/FlxBasePreloader.hx

```diff
@@ -109,8 +111,8 @@ class FlxBasePreloader extends NMEPreloader
 	 */
 	private function onEnterFrame(E:Event):Void
 	{
-		var time:Int = Lib.getTimer();
-		var min:Int = Std.int(minDisplayTime * 1000);
+		var time = Date.now().getTime() - _startTime;
```

Comment:
On Flash, Lib.getTimer() returns the time since the app was started, but on html5, it returns the current unix timestamp.